bedrock/firefox/views.py

```diff
@@ -643,8 +643,18 @@ class HelloStartView(LatestFxView):
 
 class FeedbackView(TemplateView):
 
+    donate_url = ('https://donate.mozilla.org/'
+    '?ref=EOYFR2015&utm_campaign=EOYFR2015'
+    '&utm_source=Heartbeat_survey&utm_medium=referral'
+    '&utm_content=Heartbeat_{0}stars')
+
+    valid_stars = ('3', '4', '5')
```

Comment:
Actually, we probably don't need this variable. This list is only needed in one spot, so it could just be used in place in `get_content_data`.